File: 백준/Bronze/30791. gahui and sousenkyo 1/gahui and sousenkyo 1.py
from sys import stdin

characters = list(map(int, stdin.readline().split(" ")))
top_16 = characters[0]
res = 0
for i in range(1, 5):
    if top_16 - characters[i] <= 1000:
        res += 1
    else:
        break

print(res)

File: 백준/Bronze/30791. gahui and sousenkyo 1/gahui and sousenkyo 1.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().trim().split(" ")
var res = 0;
var top_16 = parseInt(input[0]);

for(let i = 1; i <= 4; i++){
  var tmp = parseInt(input[i]);
  if(top_16 - tmp <= 1000) res += 1;
  else break;
}

console.log(res)